contracts/adapters/AaveMigrationExtension.sol

```diff
@@ -255,46 +255,25 @@ contract AaveMigrationExtension is BaseExtension, FlashLoanSimpleReceiverBase, I
     /**
      * @notice OPERATOR ONLY: Migrates a SetToken's position from an unwrapped collateral asset to another SetToken 
      * that consists solely of Aave's wrapped collateral asset
+     * @param _decodedParams The decoded migration parameters.
      * @param _underlyingLoanAmount The amount of unwrapped collateral asset to be borrowed via flash loan.
-     * @param _underlyingSupplyLiquidityAmount The amount of unwrapped collateral asset to supply for liquidity.
-     * @param _wrappedSetTokenSupplyLiquidityAmount The amount of the wrapped SetToken to supply for liquidity.
-     * @param _tokenId The Uniswap V3 position token ID related to the Extension's liquidity position.
-     * @param _exchangeName The name of the exchange to perform the trade.
-     * @param _underlyingTradeUnits The amount of the unwrapped collateral asset (in SetToken units) being sent to the exchange.
-     * @param _wrappedSetTokenTradeUnits The amount of the wrapped SetToken (in SetToken units) being received from the exchange.
-     * @param _exchangeData Additional data required for executing the trade on the specified exchange.
-     * @param _underlyingRedeemLiquidityMinAmount The minimum amount of unwrapped collateral asset to redeem from liquidity decrease.
-     * @param _wrappedSetTokenRedeemLiquidityMinAmount The minimum amount of the wrapped SetToken to redeem from liquidity decrease.
+     * @param _maxSubsidy The maximum amount of unwrapped collateral asset to be transferred to the Extension as a subsidy.
+     * @return subsidyAmount The amount of unwrapped collateral asset used by the Extension as subsidy.
      */
     function migrate(
+        DecodedParams memory _decodedParams,
         uint256 _underlyingLoanAmount,
-        uint256 _underlyingSupplyLiquidityAmount,
-        uint256 _wrappedSetTokenSupplyLiquidityAmount,
-        uint256 _tokenId,
-        string memory _exchangeName,
-        uint256 _underlyingTradeUnits,
-        uint256 _wrappedSetTokenTradeUnits,
-        bytes memory _exchangeData,
-        uint256 _underlyingRedeemLiquidityMinAmount,
-        uint256 _wrappedSetTokenRedeemLiquidityMinAmount
+        uint256 _maxSubsidy
     )
         external
         onlyOperator
+        returns (uint256 subsidyAmount)
     {
+        // Subsidize the migration
+        underlyingToken.transferFrom(msg.sender, address(this), _maxSubsidy);
```

Comment:
add support for `_maxSubsidy = 0`

Reply:
added support here [eca0526](https://github.com/IndexCoop/index-coop-smart-contracts/pull/163/commits/eca05268f6f9b903ec9c472dbde31cf00befb54c)